packages/e2e-tests/next-app-using-serverless-trace/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "next-app",
+  "version": "1.0.0",
+  "description": "Basic test Next.js app for running serverless-next.js end-to-end tests",
+  "scripts": {
+    "dev": "next",
+    "build": "next build",
+    "start": "next start -p $PORT",
+    "serverless": "serverless",
+    "e2e": "cypress run",
+    "e2e:ci": "SSG_PAGE_PATH=/ssg-page SSR_PAGE_PATH=/ssr-page APP_NAME=next-app ts-node --project tsconfig.scripts.json ../test-utils/scripts/run-e2e-ci.ts"
```

Comment:
Change `next-app` to `next-app-using-serverless-trace`. Needed so it doesn't conflict with the existing e2e tests

Reply:
Also you need to modify the CI YML file to add this to the strategy matrix